playbooks/files/rax-maas/plugins/conntrack_count.py

```diff
@@ -50,9 +63,50 @@ def get_metrics():
     return metrics
 
 
+def get_metrics_lxc_container(container=''):
+
+    # Return 0 values if we can't determine current state
+    # from a container
+    metrics = {
+        'nf_conntrack_count': {'value': 0},
+        'nf_conntrack_max': {'value': 0}}
+
+    # Create lxc container object
+    container = lxc.Container(container)
+
+    # Check if container is even running
+    try:
+        assert(container.init_pid > 1)
+        assert(container.running)
+        assert(container.state == "RUNNING")
+
+        tmpfile = os.tmpfile()
```

Comment:
Can you use `with tempfile.TemporaryFile() as tmpfile: ...` instead? That removes the need to handle `close` ourselves, along with making this compatible across versions as `os.tmpfile` was removed in Python 3.

Reply:
OK let me check. Thanks